code/game/objects/items/storage/belt.dm

```diff
@@ -748,6 +748,39 @@
 		/obj/item/ammo_casing/junk,
 	))
 
+/obj/item/storage/belt/bandolier/tactical
+	name = "tactical waist bandolier"
+	desc = "Tactical waist strap bandolier to hold 30 rounds of shotgun-sized ammo."
+	icon_state = "tactibandolier"
+	worn_icon_state = "tactibandolier"
+
+/obj/item/storage/belt/bandolier/tactical/Initialize(mapload)
+	. = ..()
+	AddElement(/datum/element/update_icon_updates_onmob)
+	atom_storage.max_slots = 30
+	atom_storage.max_total_storage = 30
+	atom_storage.do_rustle = FALSE
```

Comment:
This storage should probably be whitelisted to hold ammo only

Reply:
Wouldn't that already be true since its a bandolier subtype.

Reply:
it is